specviz/plugins/unit_change/unit_change_dialog.py

```diff
@@ -61,9 +61,9 @@ def show(self):
         self.ui.comboBox_spectral.clear()
 
         # If the units in PlotWidget are not set, do not allow the user to click the OK button
-        if not (self.hub.plot_widget.data_unit
-                and self.hub.plot_widget.spectral_axis_unit):
-            self.ui.buttonBox.button(QDialogButtonBox.Ok).setEnabled(False)
+        # if not (self.hub.plot_widget.data_unit
+        #         and self.hub.plot_widget.spectral_axis_unit):
+        #     self.ui.buttonBox.button(QDialogButtonBox.Ok).setEnabled(False)
```

Comment:
Why was this commented out?

Reply:
Addressed in the latest commit. Basically, it would disable the dialog if _either_ flux or spectral axis units were not defined.